android/app/src/main/java/org/fossasia/openevent/utils/DateUtils.java

```diff
@@ -173,4 +173,9 @@ public static String getRelativeTimeFromTimestamp(String timeStamp) throws Parse
                 (timeCreatedDate.getTime()),
                 System.currentTimeMillis(), android.text.format.DateUtils.SECOND_IN_MILLIS);
     }
+
+    public static boolean isUpcomingSession(Date start, Date end, Date current) {
```

Comment:
This is not the job of date utils, probably create a DateService. DateUtils only manages ISO8601 -> formatted date and vice versa